jobs/probe_scraper.sh

```diff
@@ -7,17 +7,25 @@ BUCKET="net-mozaws-prod-us-west-2-data-pitmo"
 CACHE_DIR="probe_cache"
 OUTPUT_DIR="probe_data"
 
+ANACONDA_PATH=/mnt/anaconda2
+ANACONDA_SCRIPT=Anaconda3-5.3.1-Linux-x86_64.sh
+
+wget --no-clobber --no-verbose -P /mnt http://repo.continuum.io/archive/$ANACONDA_SCRIPT
+rm -rf $ANACONDA_PATH
+bash /mnt/$ANACONDA_SCRIPT -b -p $ANACONDA_PATH
+rm /mnt/$ANACONDA_SCRIPT
+
 # Clone and setup the scraper
 git clone https://github.com/mozilla/probe-scraper.git
 cd probe-scraper
-python setup.py bdist_egg
-pip install -r requirements.txt
+python3 setup.py bdist_egg
```

Comment:
At this point `python3` is just a `python` alias, but it does make it obvious what we're using.